Maestro/subscriptions.json

```diff
@@ -640,7 +640,10 @@
         "https://github.com/dotnet/versions/blob/master/build-info/docker/debian/jessie.txt",
         "https://github.com/dotnet/versions/blob/master/build-info/docker/debian/stretch.txt"
       ],
-      "action": "dotnet-docker-nightly-pipebuild"
+      "action": "dotnet-docker-nightly-pipebuild",
+      "actionArguments": {
+        "vsoSourceBranch": "master"
```

Comment:
What defaulting are you referring to?  The $(Build.SourceBranch) variable is empty in the dotnet-docker-nightly-pipebuild which causes pipebuild to throw and ArgEx.  Maestro has no defaulting and it appears vso doesn't either at the variable level.  Going out on a limb here that this may not cause an issue for the built in `Get Sources` because it may have defaulting logic.